src/main/kotlin/no/nav/syfo/Bootstrap.kt

```diff
@@ -1,179 +1,107 @@
 package no.nav.syfo
 
-import com.fasterxml.jackson.databind.DeserializationFeature
-import com.fasterxml.jackson.databind.ObjectMapper
-import com.fasterxml.jackson.databind.SerializationFeature
-import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
-import com.fasterxml.jackson.module.kotlin.registerKotlinModule
-import com.ibm.msg.client.jms.JmsConstants.USER_AUTHENTICATION_MQCSP
-import io.ktor.client.*
-import io.ktor.client.engine.apache.*
-import io.ktor.client.features.json.*
-import io.ktor.util.*
-import io.prometheus.client.hotspot.DefaultExports
+import io.ktor.application.*
+import io.ktor.routing.*
+import io.ktor.server.engine.*
+import io.ktor.server.netty.*
 import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.GlobalScope
 import kotlinx.coroutines.Job
 import kotlinx.coroutines.launch
 import no.nav.emottak.subscription.SubscriptionPort
-import no.nav.syfo.application.ApplicationServer
 import no.nav.syfo.application.ApplicationState
 import no.nav.syfo.application.BlockingApplicationRunner
-import no.nav.syfo.application.createApplicationEngine
-import no.nav.syfo.client.*
-import no.nav.syfo.client.azuread.v2.AzureAdV2Client
+import no.nav.syfo.application.api.registerNaisApi
 import no.nav.syfo.db.Database
 import no.nav.syfo.db.VaultCredentialService
 import no.nav.syfo.kafka.*
-import no.nav.syfo.services.JournalService
-import no.nav.syfo.services.RuleService
-import no.nav.syfo.services.SignerendeLegeService
 import no.nav.syfo.util.*
 import no.nav.syfo.vault.RenewVaultService
 import no.nav.syfo.ws.createPort
 import org.apache.cxf.ws.addressing.WSAddressingFeature
-import org.apache.http.impl.conn.SystemDefaultRoutePlanner
 import org.apache.kafka.clients.producer.KafkaProducer
 import org.slf4j.Logger
 import org.slf4j.LoggerFactory
-import java.net.ProxySelector
+import java.util.concurrent.TimeUnit
 import javax.jms.Session
 
-val objectMapper: ObjectMapper = ObjectMapper()
-    .registerModule(JavaTimeModule())
-    .registerKotlinModule()
-    .configure(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS, false)
+val logger: Logger = LoggerFactory.getLogger("no.nav.syfo.padm2")
```

Comment:
Er det noen spesiell grunn til at den nå heter `logger` i stedet for `log`? 🤷‍♂️

Reply:
> Er det noen spesiell grunn til at den nå heter `logger` i stedet for `log`? 🤷‍♂️

I ktor-initialiseringen likte ikke kompilatoren log=log, så da ble det log=logger istedet.